tests/testStatsdClient.cpp

```diff
@@ -118,14 +118,24 @@ void testSendRecv(uint64_t batchSize) {
         throwOnError(client);
         expected.emplace_back("sendRecv.myTiming:2|ms|@0.10");
 
-        // Send a metric explicitly
-        client.send("tutu", 241, "c", 2.0f);
+        // Send a set with 1227 total uniques
+        client.set("tutu", 1227, 2.0f);
         throwOnError(client);
-        expected.emplace_back("sendRecv.tutu:241|c");
+        expected.emplace_back("sendRecv.tutu:1227|s");
+
+        // Gauge but with tags
+        client.gauge("dr.rösti.grabe", 333, 1.f, {"liegt", "im", "weste"});
```

Comment:
:smile:

Reply:
i thought it would be a funny easter egg in the tests since my ancestors come from the bodensee area near bischofszell. we still speak an old alemannisch dialect here but i visit switzerland as often as i can :smile: